node-playwright/Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM mcr.microsoft.com/playwright:focal
+ARG PLAYWRIGHT_VERSION=1.33.0
```

Comment:
i guess we don't want the hardcoded default here? or if it could be dynamically resolved to latest version, that would be fine